app/controllers/searches_controller.rb

```diff
@@ -1,6 +1,20 @@
 class SearchesController < ApplicationController
-    def index
-        @places = SearchLogic.get_places_from_city_string(params[:city_name], params[:place_type])
-        render params[:place_type]
+  def index
+    if params[:city_name].present?
+      selected_place_types = []
+      selected_place_types << 'attractions' if params[:place_type_attractions] == 'attractions'
+      selected_place_types << 'hotels' if params[:place_type_hotels] == 'hotels'
+      selected_place_types << 'restaurants' if params[:place_type_restaurants] == 'restaurants'
+
+      @places = selected_place_types.flat_map do |place_type|
+        SearchLogic.get_places_from_city_string(params[:city_name], place_type)
+      end
+
+      render :index
```

Comment:
It's neat that you were able to get all three place types on one template. I guess all the columns that we care about on the list view are common between the three place types, so that's nice.